package.json

```diff
@@ -123,10 +124,5 @@
     "stylelint-order": "6.0.4",
     "stylelint-selector-bem-pattern": "3.0.1",
     "user-agent-data-types": "0.4.2"
-  },
-  "overrides": {
-    "stylelint-selector-bem-pattern": {
-      "stylelint": "16.1.0"
-    }
```

Comment:
surprised to see this removed and not causing linting issues - i didn't think it was resolved yet 🤷 

https://github.com/simonsmith/stylelint-selector-bem-pattern/pull/68

Reply:
IDK either 🤷